flink-runtime/src/test/java/org/apache/flink/runtime/state/InternalPriorityQueueTestBase.java

```diff
@@ -136,7 +136,7 @@ void testPeekPollOrder() {
             --lastSize;
         }
 
-        assertThat(priorityQueue.isEmpty()).isTrue();
+        assertThat(priorityQueue);
```

Comment:
Right now this is ignored, I think unintentionally. :)

Reply:
Thanks @ferenc-csaky for catching this.
Fixed.